concordia/templates/transcriptions/asset_detail.html

```diff
@@ -292,11 +292,12 @@ <h2>Tags</h2>
             </button>
             <div>
                 <ul>
-                    <li>Type what you see. Preserve original spelling, punctuation, and line breaks.</li>
-                    <li>Use brackets [ ] around deleted, illegible or partially legible text.</li>
-                    <li>Transcribe any words or letters you can't identify as ?.</li>
-                    <li>Use square brackets and asterisks around text from margins [ * * ].</li>
+                    <li>Transcribe original spelling, punctuation, word order, and any page numbers or catalog marks.</li>
+                    <li>Preserve line breaks except when a word breaks over a line or page. Then transcribe it on the line or page where it starts.</li>
+                    <li>Use brackets [ ] around deleted, illegible or partially legible text, and square brackets and asterisks around text in margins [ * * ].</li>
+                    <li>Transcribe any words or letters you can't identify as [?].</li>
                     <li>Include insertions where you would read them in the text.</li>
+                    <li>Visit the Help Center for more detailed instructions.</li>
```

Comment:
Do you want this bullet point to have the link to the help center (i.e. `<a href="{% url 'help-center' %}">Help Center</a>`) and take out the text below ("Find more detailed instructions...")?

Reply:
Lets just remove that bullet because there is text right below the bullets that says, ` Find more detailed instructions in the <a href="{% url 'help-center' %}">Help Center</a>`